icgcget/cli.py

```diff
@@ -124,45 +130,47 @@ def download(ctx, file_ids, repos, manifest, output,
     old_session_info = None
     if os.path.isfile(pickle_path):
         old_session_info = pickle.load(open(pickle_path, 'r+'))
-        if psutil.pid_exists(old_session_info['pid']):
-            raise click.Abort("Download currently in progress")
-    if file_ids == 'resume':
+    if ids == 'resume':
         session_info = old_session_info
     else:
-        validate_ids(file_ids, manifest)
-        session_info = dispatch.download_manifest(repos, file_ids, manifest, output, yes_to_all, api_url)
+        validate_ids(ids, manifest)
+        session_info = dispatch.download_manifest(repos, ids, manifest, output, api_url, no_ssl_verify)
     if old_session_info:
-        session_info['object_ids'] = compare_ids(session_info['object_ids'], old_session_info['object_ids'], yes_to_all)
-    pickle.dump(session_info, open(pickle_path, 'w'), pickle.HIGHEST_PROTOCOL)
-    dispatch.download(session_info['object_ids'], staging, output,
-                      cghub_access, cghub_path, cghub_transport_parallel,
-                      ega_access, ega_path, ega_transport_parallel, ega_udt,
-                      gdc_access, gdc_path, gdc_transport_parallel, gdc_udt,
-                      icgc_access, icgc_path, icgc_transport_file_from, icgc_transport_parallel,
-                      pdc_access, pdc_path, pdc_region, pdc_transport_parallel)
+        session_info['object_ids'] = compare_ids(session_info['object_ids'], old_session_info['object_ids'], override)
+    pickle.dump(session_info, open(pickle_path, 'w', 0777), pickle.HIGHEST_PROTOCOL)
```

Comment:
For hackability, debuggability, etc., it would probably be better not to use pickle, but use a text format such as json.